generated/nidcpower/session.py

```diff
@@ -2378,6 +2378,55 @@ def fetch_multiple(self, count, timeout=datetime.timedelta(seconds=1.0)):
         the *NI DC Power Supplies and SMUs Help* for more information about
         configuring this method.
 
+        Note: This method is not supported on all devices. Refer to `Supported Methods by Device <REPLACE_DRIVER_SPECIFIC_URL_2(nidcpowercref.chm, supportedfunctions)>`__ for more information about supported devices.
+
+        Tip:
+        This method requires repeated capabilities (usually channels). If called directly on the
+        nidcpower.Session object, then the method will use all repeated capabilities in the session.
+        You can specify a subset of repeated capabilities using the Python index notation on an
+        nidcpower.Session instance, and calling this method on the result.:
+
+            session.channels['0,1'].fetch_multiple(count, timeout=datetime.timedelta(seconds=1.0))
+
+        Args:
+            count (int): Specifies the number of measurements to fetch.
+
+            timeout (datetime.timedelta): Specifies the maximum time allowed for this method to complete, in seconds. If the method does not complete within this time interval, NI-DCPower returns an error.
+
+                Note: When setting the timeout interval, ensure you take into account any triggers so that the timeout interval is long enough for your application.
+
+
+        Returns:
+            measurements (list of float): Returns an array of named tuples.
+
+        '''
+        import collections
+        Measurements = collections.namedtuple('Measurements', ['voltage', 'current', 'in_compliance'])
+
+        voltage_measurements, current_measurements, in_compliance = self._fetch_multiple(count, timeout)
+
+        measurements = []
+        for i in range(count):
```

Comment:
Better:

    for i in zip(self._fetch_multiple(count, timeout)):
        measurements.append(Measurement(i))

I bet this can be a clean one-liner too:

    return [ ... ]

Reply:
Odd. This worked for me:

    >>> a = array.array('i',[1,2,3])
    >>> b = [1,2,3]
    >>> zip (a,b)
    [(1, 1), (2, 2), (3, 3)]

Reply:
When I use the named tuple I get this:
```
>   return [Measurement(*m) for m in zip(self._fetch_multiple(count, timeout))]
E   TypeError: __new__() missing 2 required positional arguments: 'current' and 'in_compliance'
```

It doesn't matter if I use `Measurement(m)` or `Measurement(*m)`

Reply:
Was that code executed in Python 2 or 3?